packages/adblocker/src/filters/interface.ts

```diff
@@ -14,4 +14,5 @@ export default interface IFilter {
   getTokens: () => Uint32Array[];
   serialize: (buffer: StaticDataView) => void;
   getSerializedSize(compression: boolean): number;
+  toString(): string;
```

Comment:
why this is needed?

Reply:
Updated. This change was for the test file.